query-node/mappings/src/forum.ts

```diff
@@ -83,10 +84,19 @@ async function getPost(store: DatabaseManager, postId: string, relations?: 'thre
 }
 
 async function getActorWorker(store: DatabaseManager, actor: PrivilegedActor): Promise<Worker> {
+  const workingGroup = await store.get(WorkingGroup, {
+    where: { id: 'forumWorkingGroup'},
+    relations: ['leader']
+  })
+
+  if (!workingGroup) {
+    throw new Error(`Forum Working Group not found!`)
+  }
+  
   const worker = await store.get(Worker, {
     where: {
       group: { id: 'forumWorkingGroup' },
-      ...(actor.isLead ? { isLead: true } : { runtimeId: actor.asModerator.toNumber() }),
+      ...(actor.isLead ? { id: workingGroup.leader?.id } : { runtimeId: actor.asModerator.toNumber() }),
     },
```

Comment:
Deploying new mappings usually means remapping the QN. In this case `isLead` will have been fixed by the changes in  `query-node/mappings/src/workingGroups.ts`. So the way I understand it, this changes (in `query-node/mappings/src/forum.ts`) are not necessary. Or am I missing something ?